client/src/App.js

```diff
@@ -1,23 +1,13 @@
 import React from 'react';
 import './App.css';
-
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
lets keep this consistent and use single quotes (like the line above), and also alphabetize the imports.

(i'll try to add some something to lint this in the PR CI)